SoftLayer/CLI/user/grant_access.py

```diff
@@ -0,0 +1,44 @@
+"""User grant access to devices."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+import SoftLayer
+
+from SoftLayer.CLI.command import SLCommand as SLCommand
+from SoftLayer.CLI import environment
+
+
+@click.command(cls=SLCommand, )
+@click.argument('identifier')
+@click.option('--hardware', help="Hardware ID")
+@click.option('--virtual', help="Virtual Guest ID")
+@click.option('--dedicated', help="dedicated host ID")
+@environment.pass_env
+def cli(env, identifier, hardware, virtual, dedicated):
+    """Grant access from a user to an specific device.
+
+    Example: slcli user grant-access 123456 --hardware 123456789
+    """
+
+    mgr = SoftLayer.UserManager(env.client)
+    result = False
+    if hardware:
+        result = mgr.grant_hardware_access(identifier, hardware)
+        if result:
+            click.secho("Grant to access to hardware: %s" % hardware, fg='green')
```

Comment:
```suggestion
            click.secho(f"User {identifier}  has been given access to hardware {hardware} ", fg='green')
```

I'd like to start using [f strings](https://fstring.help) anything we need variable substitution now that we only support py3.7+.  And the message needed to be cleaned up a little.

Use the same format on the other 2 messages below as well.

Reply:
The messages were updated using `f strings`